docs/publishers/events.rst

```diff
@@ -0,0 +1,50 @@
+Custom DOM events
+=================
+
+Hypothesis client emits a few custom DOM events to which your app can react.
```

Comment:
The documentation doesn't specify which target DOM events are dispatched at, although the code example below indicates the document body. I suggest we just specify the document body, although internally this is abstracted from the `Guest` code via the `Guest.element` property.